G_channels.md

```diff
@@ -249,6 +249,24 @@ socket = assign(socket, :user, msg["user"])
 
 Sockets store assigned values as a map in `socket.assigns`.
 
+#### Failure Semantics
+
+Servers will restart & clients will fail - understanding how Phoenix responds to these events and the guarantees it offers will  help you design robust systems. 
+
+- Subscriptions
+ 
+On the server, subscriptions are held in an ETS table (which resides in memory). If a server restarts or crashes, the subscriptions will need to be recreated for clients to receive updaes. Fortunately, this functionality is built into the Phoenix javascript client. The clients will attempt to reconnect to the server. Once they do, they'll rejoin the channels they were subscribed to using exponential back off.
```

Comment:
Small typos: 
- "updaes" => "updates"
- "javascript" => "JavaScript"

I also think that the "using exponential back off" should be taken from the last sentence and attached to the sentence before it.
The clients will attempt to reconnect to the server with an exponential back off. Once they do, they'll rejoin the channels they were subscribed to.